Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -0,0 +1,391 @@
+//
+//  Created by Dan Federman on 11/9/19.
+//  Copyright © 2019 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import XCTest
+
+@testable import CacheAdvance
+
+final class CacheAdvanceTests: XCTestCase {
+
+    // MARK: XCTestCase
+
+    override func setUp() {
+        super.setUp()
+        FileManager.default.createFile(atPath: testFileLocation.path, contents: nil, attributes: nil)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        try? FileManager.default.removeItem(at: testFileLocation)
+    }
+
+    // MARK: Behavior Tests
+
+    func test_append_singleMessageThatFitsCanBeRetrieved() throws {
```

Comment:
I'd try to break this method name up a bit more.
Maybe something like: `test_append_singleMessageThatFits_canBeRetrieved()`
Then the next one can be: `test_append_singleMessageThatDoesNotFit_fails()`

I generally think of naming my tests methods in this format:

`test_<methodName>_<context>_<expectedBehavior>()`

Reply:
I don’t normally split out context... but I like it.